labonneboite/tests/web/integration/test_pro_version.py

```diff
@@ -10,24 +13,62 @@ def test_user_is_pro(self):
         """
         Test that the Pro Version is correctly detected in various cases.
         """
+        self.assertIn('@pole-emploi.fr', settings.VERSION_PRO_ALLOWED_EMAIL_SUFFIXES)
 
         user_pro = User.create(email=u'john.doe@pole-emploi.fr', gender=u'male', first_name=u'John', last_name=u'Doe')
         user_public = User.create(email=u'john.doe@gmail.com', gender=u'male', first_name=u'John', last_name=u'Doe')
 
-#        FIXME restore when version PRO will be back
-#        with self.test_request_context:
-#
-#            # Pro Version should be disabled for non logged users.
-#            self.assertFalse(util.user_is_pro())
-#
-#            # Pro Version should be enabled for a user with a PRO email.
-#            self.login(user_pro)
-#            self.assertTrue(util.user_is_pro())
-#            self.logout()
-#            self.assertFalse(util.user_is_pro())
-#
-#            # Pro Version should be disabled for a user with a non PRO email.
-#            self.login(user_public)
-#            self.assertFalse(util.user_is_pro())
-#            self.logout()
-#            self.assertFalse(util.user_is_pro())
+        with self.test_request_context:
+
+            # Pro Version should be disabled for non logged users.
```

Comment:
User which is not logged in should be considered not pro of course.

Reply:
J'ai rajouté une note pour préciser que les tests par IP n'étaient pas possibles et que du coup, seuls les tests par courriel sont faits.
Cette ligne peut prêter confusion sur la détection des utilisateurs pro.